pymc3/tests/test_distributions_random.py

```diff
@@ -13,16 +13,18 @@
 from .test_distributions import (
     build_model, Domain, product, R, Rplus, Rplusbig, Rplusdunif,
     Unit, Nat, NatSmall, I, Simplex, Vector, PdMatrix,
-    PdMatrixChol, PdMatrixCholUpper, RealMatrix
+    PdMatrixChol, PdMatrixCholUpper, RealMatrix, RandomPdMatrix
 )
 
 
 def pymc3_random(dist, paramdomains, ref_rand, valuedomain=Domain([0]),
-                 size=10000, alpha=0.05, fails=10, extra_args=None):
+                 size=10000, alpha=0.05, fails=10, extra_args=None,
+                 model_args={}):
```

Comment:
Having `model_args={}` is making travis unhappy, because [of this].(https://stackoverflow.com/questions/1367883/python-methods-default-parameter-values-are-evaluated-once)

Better to do `model_args=None` and give it an empty dict if it is None.

Reply:
Fixing occurances of this will get rid of the little (!) that shows up next to the travis builds